3D_Imaging/CellCycleMaps/README.md

```diff
@@ -24,5 +20,6 @@ At the first run, the script will split the dataset into train/val/test splits a
 ```bash
 python runModel.py -dataDir DATADIR -m test -testCSV ./models/test_1ch.csv -arch custom_cnn -exp CNN_exp1_merged_data_ch1
```

Comment:
would add a sentence here as in '`A new folder "Results" will be generated containing ...`'